sites/docs/src/content/docs/tutorials/google_colab/nf-core-colab-guide.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Running nf-core Pipelines on Google Colab"
+subtitle: "A guide to running and interacting with nf-core pipelines using Colab and VS Code"
+---
+
+# Introduction
+
+Running nf-core pipelines can be computationally intensive, requiring resources not easily available to students, newcomers, or participants in hands-on training workshops.
+This tutorial shows two ways you can use Google Colab to run nf-core pipelines entirely in the cloud, either in a browser-based execution or connected to VS Code via the vscode-colab package for a full development experience.
+While Colab has limitations, such as session timeouts and a lack of root access, it offers a free and accessible platform ideal for learning, teaching, and prototyping workflows in resource-constrained environments.
+Also make sure to checkout the [blogpost](https://nf-co.re/blog/2025/nf-core-colab-guide) on this topic for deeper insights and more tips for running nf-core pipelines using Colab.
+
+# Setting up the environment in Google Colab
+
+After opening a Colab notebook with the machine type of your choice, the first step is to install Nextflow and it's dependencies. Since Colab does not provide root access, you will need to install Nextflow in the user's home directory.
```

Comment:
Before this maybe point to some docs from Google on how to register and get stetup ?